setup.cfg

```diff
@@ -0,0 +1,7 @@
+[versioneer]
+VCS = git
+style = pep440
+versionfile_source = pyfftw/_version.py
+versionfile_build = pyfftw/_version.py
+tag_prefix =
```

Comment:
This probably needs to be `v` as the tags here seem to be also prefixing with `v`. Also I think one needs to rerun `versioneer install` after changing this file.